src/components/shell-panel/shell-panel.scss

```diff
@@ -35,7 +35,6 @@
   absolute
   bottom-0
   top-0
-  z-10
```

Comment:
Do we know why some of these are here? Just don't want to cause any regressions. Maybe more testing with the ones we remove?

Reply:
I couldn't deduce the need for it, and didn't notice any visual issues after removing. 

> Maybe more testing with the ones we remove?

Great idea.